src/Input/TextField.js

```diff
@@ -47,6 +50,11 @@ class TextField extends React.Component {
     multiLine: React.PropTypes.bool,
 
     /**
+     * Button search for text field
+     */
+    searchButton: React.PropTypes.bool,
```

Comment:
I think, instead of adding searchButton here just add setOnFocus prop or something like that. That way, the textfield will know that it will be set onfocus